collection_templates/islandora__bookCollection.xml

```diff
@@ -1,10 +1,10 @@
 <mods xmlns="http://www.loc.gov/mods/v3" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="3.5" xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-5.xsd">
-    <identifier type="filename">HikingClubHandbook_DATE</identifier>
+    <identifier type="filename">HikingClubHandbook_ | ${mods_dateIssued}</identifier>
```

Comment:
Do you really want the exact string "HikingClubHandbook_ |" as part of the identifier? For example, it looks like it would result in the identifier 
 filename would look like: "HikingClubHandbook_ | 2017".

Reply:
Is there a change there might be more than one each year?

Reply:
@DonRichards - Hi Don. No, I do not want the pipe in the identifier. I think initially when we talked you said you needed this to separate the variable from blanket string values. I've included a pipe through the XML, so let me know if this is not the case (and the variable alone is sufficient) and I'll remove the pipes.

So far in the history of the hiking club handbook, there has only been one year associated with each item. While it's always possible that this may change in the future, I do not think it would likely ever happen.

Reply:
I would suggest against it but it shouldn't hurt anything, it's only the file name.

Reply:
@DonRichards - I've removed the pipes. We may also want to consider renaming these files, as they'll be collection-specific, for clarity.